contracts/Vault.sol

```diff
@@ -123,21 +130,27 @@ contract Vault is ExchangeRate, Initializable, OwnableUpgradeSafe {
             aTokenAmount = balance;
         }
 
+        require(aTokenAmount > 0, "amount to withdraw cannot be zero");
+        bToken.burn(address(this), balance);
+        require(aTokenAmount >= depositorToPrincipal[msg.sender], "atoken must be more than depositorPrincipal");
+        require(depositorReserve >= SafeMath.sub(aTokenAmount, depositorToPrincipal[msg.sender]), "atoken must be more than depositorPrincipal");
```

Comment:
doesn't `aTokenAmount.sub(depositorToPrincipal[msg.sender])` achieve the same thing as `SafeMath.sub(aTokenAmount, depositorToPrincipal[msg.sender])` ?

might make things easier to read. Instead of
`depositorToPrincipal[msg.sender] = SafeMath.add(depositorToPrincipal[msg.sender], atokenAmount);`

We'll have 
`depositorToPrincipal[msg.sender] = depositorToPrincipal[msg.sender].add(atokenAmount)`

Reply:
good find, I did think about changing them all to the latter `depositorToPrincipal[msg.sender] = depositorToPrincipal[msg.sender].add(atokenAmount)`, will do this